packages/node_modules/@ciscospark/widget-roster/src/components/add-participant/index.js

```diff
@@ -74,7 +74,7 @@ function AddParticipant({
     }
     else if (searchResults.results && searchResults.results.length) {
       results = (
-        <div className={classNames(`${baseClassPrefix}-results`, styles.results)}>
+        <div style={{height: '100%', width: '100%'}} className={classNames(`${baseClassPrefix}-results`, styles.results)}>
```

Comment:
Should these inline styles be added to the css file instead?

Reply:
The autosizer for virtualized doesn't seem to like actual css styles, but only react ones. I had to do the same for recents.